mk/config.mk

```diff
@@ -226,3 +226,6 @@ ifneq ($(CFG_TA_GPROF_SUPPORT),y)
 $(error Cannot instrument user libraries if user mode profiling is disabled)
 endif
 endif
+
+# Enable Secure Data Path support in OP-TEE core
```

Comment:
I think the best is to explains basics and constraints of SDP in the documentation/extensions/.
I fear explaining SDP enable/disable in this config.mk file will either be to long or not short.
I will see what could best fit and propose a change for this comment.